.scalafmt.conf

```diff
@@ -1,3 +1,4 @@
+version = "2.0.0-RC4"
```

Comment:
We specify scalafmt version in the boostrap script instead https://github.com/scala-native/scala-native/blob/master/scripts/scalafmt#L6

I'd avoid combining scalafmt upgrades with any other changes. It usually produces a giant diff that obfuscates any non-formatting changes.

Reply:
This change is done by Metals automatically, as format on save requires a scalafmt version in the scalafmt config (I don't use format on save, so we can remove this diff altogether).

Reply:
Lets drop it for this pr.